### PR TITLE
SQL - Update

### PR DESCRIPTION
**Pull request by kapnobatai137**
- sql update discussion insight

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| update-discussion-insight | <ul><li>`Playground` pointing to `https://student.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2Fa41bb442321200a0ded3475100e5f9a9%2Fraw%2Ff7f2ef8a49bd551c39b5fa2a8255d62a72ff1f73%2Fsql-update-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| update-discussion-insight | <ul><li>`Playground` pointing to `https://student.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2Fa41bb442321200a0ded3475100e5f9a9%2Fraw%2Ff7f2ef8a49bd551c39b5fa2a8255d62a72ff1f73%2Fsql-update-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: sql/dml/update/update-discussion-insight.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+# Test Yourself
+
+---
+
+## Content
+
+Let's see how you can handle manipulating data!
+
+Consider the following table `movies` table:
```
Comment (Stefan-Stojanovic) [inline sql/dml/update/update-discussion-insight.md:25]:
```suggestion
Consider the following `movies` table:
```

**Review by Stefan-Stojanovic — CHANGES_REQUESTED** [review]

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| update-discussion-insight | <ul><li>`Playground` pointing to `https://learner.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2Fa41bb442321200a0ded3475100e5f9a9%2Fraw%2F000ef19e1786ccc4ca5f846ad98ef2b27b38c189%2Fsql-update-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| update-discussion-insight | <ul><li>`Playground` pointing to `https://learner.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2Fa41bb442321200a0ded3475100e5f9a9%2Fraw%2F000ef19e1786ccc4ca5f846ad98ef2b27b38c189%2Fsql-update-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| update-discussion-insight | <ul><li>`Playground` pointing to `https://learner.enkicamp.com/public/playground?jsonUrl=https%3A%2F%2Fgist.githubusercontent.com%2Fkapnobatai137%2Fa41bb442321200a0ded3475100e5f9a9%2Fraw%2F000ef19e1786ccc4ca5f846ad98ef2b27b38c189%2Fsql-update-discussion-insight.json` is broken because `HTTP_404`</li></ul> |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Pull request merged**